Libplanet/Net/Swarm.cs

```diff
@@ -1746,15 +1752,9 @@ await GetBlockHashesAsync(peer, locator, stop, cancellationToken)
                     // same genesis block...
                     else if (!BlockChain.ContainsBlock(branchPoint))
                     {
-                        // Create a whole new chain because the branch point doesn't exist on
-                        // the current chain.
-                        _logger.Debug("Create new chain...");
-                        workspace = new BlockChain<T>(
-                            workspace.Policy,
-                            workspace.Store,
-                            Guid.NewGuid()
-                        );
-                        scope.Add(workspace.Id);
+                        _logger.Debug(
+                            "Reorganization for the genesis block isn't allowed in this chain.");
```

Comment:
How about this?

```
Since the genesis block is fixed to {BlockChain.Genesis} protocol-wise, the blockchain which does not share any mutual block is not acceptable.
```